modules/api/src/main/java/com/intuit/wasabi/api/HttpHeader.java

```diff
@@ -59,8 +60,9 @@ public ResponseBuilder headers(int status) {
                 .cacheControl(cacheControl)
                 .header(ACCESS_CONTROL_ALLOW_ORIGIN, "*")
                 .header(ACCESS_CONTROL_ALLOW_HEADERS, "Authorization,X-Forwarded-For,Accept-Language,Content-Type")
-                .header(ACCESS_CONTROL_ALLOW_METHODS, "GET,POST,OPTIONS")
-                .header(ACCESS_CONTROL_REQUEST_METHOD, "GET,POST,OPTIONS")
+                .header(ACCESS_CONTROL_ALLOW_METHODS, "GET,POST,PUT,DELETE,OPTIONS")
+                .header(ACCESS_CONTROL_REQUEST_METHOD, "GET,POST,PUT,DELETE,OPTIONS")
```

Comment:
@longdogz just to be consistent. because non OPTIONS (POST, GET) returns that as well.